assets/template-giftcard.css

```diff
@@ -22,7 +22,7 @@ h6:empty {
 }
 
 html {
```

Comment:
When I adjust the font size in the gift card page I get a Liquid internal error:

https://screenshot.click/05-09-ciwss-eoc1q.mp4